code/modules/modular_computers/file_system/programs/ai_restorer.dm

```diff
@@ -0,0 +1,117 @@
+/datum/computer_file/program/aidiag
```

Comment:
afaik this computer never served a purpose when it was standalone, i'm not sure what the point in making a program of it was